cfgov/tccp/filterset.py

```diff
@@ -12,21 +12,20 @@
 
 
 class CardSurveyDataFilterSet(filters.FilterSet):
-    targeted_credit_tiers = filters.ChoiceFilter(
+    credit_tier = filters.ChoiceFilter(
         choices=CreditTierChoices[1:],
-        method="filter_targeted_credit_tiers",
+        method="filter_credit_tier",
         label="Your credit score",
         initial=CreditTierChoices[2][0],
         null_label=None,
         empty_label=None,
         widget=Select,
     )
-    geo_availability = filters.ChoiceFilter(
+    location = filters.ChoiceFilter(
```

Comment:
Not for this PR, but we should come back once this form is finalized and add our [typical form validation](https://cfpb.github.io/design-system/components/alerts#validation) since all the fields on the landing page are required. Maybe we can just mark this as a TODO for now?

Reply:
Sure, although as of #8236, the situation on the landing page is no longer required. Which fields did you have in mind? Right now everything is either a checkbox or a dropdown so there's no way for users to implement invalid values for any of the fields.

If a crafty user does construct an invalid URL, by passing an invalid value to the query string, the list view won't currently show any validation errors, but will show an empty list and an error message, [for example](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/?targeted_credit_tiers=invalid):

<img width="1212" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/7fc39b4b-1388-46a4-9db6-7a00566c7f27">